controllers/notifications.go

```diff
@@ -0,0 +1,87 @@
+package controllers
+
+import (
+	"badi-cal/badi-events-backend/models"
+	"badi-cal/badi-events-backend/orm"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+type NotificationPayload struct {
+	ID             uint     `json:"id"`
+	Event_id       string   `json:"event_id"`
+	Body           string   `json:"body"`
+	Scheduled_time JSONTime `json:"scheduled_time"`
+	Status         string   `json:"status"`
+	Attempts       int      `json:"attempts"`
+}
+
+var statusMap = map[int]string{
+	0: "unsent",
+	1: "sent",
+	2: "failed",
+}
+
+func StatusString(status orm.NotificationStatus) string {
+	return statusMap[int(status)]
+}
+
+func toNotificationPayload(notification orm.Notification) NotificationPayload {
+	return NotificationPayload{
+		ID:             notification.ID,
+		Event_id:       notification.Event_id,
+		Body:           notification.Body,
+		Scheduled_time: JSONTime(notification.Scheduled_time),
+		Status:         StatusString(notification.Status),
+		Attempts:       notification.Attempts,
+	}
+}
+
+func NotificationsCreate(w http.ResponseWriter, r *http.Request, app *App) {
+	body, _ := ioutil.ReadAll(r.Body)
+	var data models.NotificationCreatePayload
+	json.Unmarshal([]byte(body), &data)
+	b, _ := json.Marshal(data)
```

Comment:
I am not understanding what we are doing with this line.  It seems like we are taking the data that just got unmarshalled from JSON and marshalling it back into JSON, but not doing anything with its value and just ignoring its error value.  Are we just trying to test out that we can marshal the data back into JSON after we unmarshalled it from JSON?

Reply:
lol, sorry, that was just some test output I forgot to remove

Reply:
Fixed.